app/blueprints/round/routes.py

```diff
@@ -145,7 +146,7 @@ def edit_round(round_id):
     return render_template("round.html", **params, error=error)
 
 
-@round_bp.route("/<round_id>/clone")
+@round_bp.route("/<round_id>/grant/<fund_id>/clone")
 def clone_round(round_id, fund_id):
```

Comment:
This should not be a GET request, it should be a POST request. Anything that mutates our DB (create/edit/delete) should be behind a form that gets CSRF protection.

You'll need to update all of the places that call this endpoint to using a <form> element please 🙏